rootex/common/common.h

```diff
@@ -33,4 +33,35 @@
 	{                                                                              \
 		String msg = String(__FILE__) + ":" + String(__FUNCTION__) + ": " + m_Msg; \
 		::OS::PrintError(String(m_file) + ":" + String(m_func) + ": " + m_Msg);    \
+	}
+
+/// Logs function, message in white color, doesn't log in editor console
+#define PRINT_SILENT(m_Msg)                                      \
+	{                                                     \
+		String msg = String(__FUNCTION__) + ": " + m_Msg; \
+		::OS::PrintSilent(msg);                                 \
+	}
+/// Logs file, line, function, message in yellow color, doesn't log in editor console
+#define WARN_SILENT(m_Msg)                                                                                                 \
```

Comment:
Press `Tab` on these once so that VS corrects the formatting of the `\` character